tests/cbmc/sources/make_common_datastructures.c

```diff
@@ -77,7 +77,17 @@ const char *nondet_c_str_is_allocated(size_t max_size)
 struct s2n_stuffer_reservation *cbmc_allocate_s2n_stuffer_reservation()
 {
     struct s2n_stuffer_reservation *reservation = can_fail_malloc(sizeof(*reservation));
-    if (reservation != NULL) { reservation->stuffer = cbmc_allocate_s2n_stuffer(); }
+    if (reservation != NULL) {
+        __CPROVER_assume(reservation->write_cursor <= UINT32_MAX - reservation->length);
```

Comment:
Could you add a comment here?

Reply:
Moved this logic to the corresponding is_valid function